src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,69 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
+import { addComment } from '../api/dataFromServer';
+import { Comment } from '../types';
+
+interface Props {
+  postId: number;
+  onChangeComment: (newComment: Comment) => void;
+}
+
+export const NewCommentForm: React.FC<Props> = ({
+  postId,
+  onChangeComment,
+}) => {
+  const [title, setTitle] = useState('');
+  const [email, setEmail] = useState('');
+  const [message, setMessage] = useState('');
```

Comment:
you can combine these fields in one object & make 1 function to handle set values
But it's up to you
```suggestion
  const [fields, setFields] = useState({ title: '', email: '', message: '' });
```